backend/package.json

```diff
@@ -1,42 +1,43 @@
 {
-  "name": "backend",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "type": "module",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon server.js",
-    "lint": "eslint ../backend --ext js,jsx --report-unused-disable-directives --max-warnings 0"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "bcrypt": "^5.1.0",
-    "connect-session-sequelize": "^7.1.7",
-    "cookie-parser": "^1.4.6",
-    "cors": "^2.8.5",
-    "dotenv": "^16.3.1",
-    "express": "^4.18.2",
-    "express-session": "^1.17.3",
-    "morgan": "^1.10.0",
-    "nodemon": "^2.0.22",
-    "pg": "^8.11.1",
-    "pg-hstore": "^2.3.4",
-    "sequelize": "^6.32.1",
-    "socket.io": "^4.7.1"
-  },
-  "devDependencies": {
-    "eslint": "^8.46.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-prettier": "^8.9.0",
-    "eslint-plugin-react": "^7.33.0",
-    "prettier": "^3.0.0"
-  },
-  "prettier": {
-    "tabWidth": 4,
-    "useTabs": false,
-    "trailingComma": "none"
-  }
+    "name": "backend",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "nodemon server.js",
+        "lint": "eslint ../backend --ext js,jsx --report-unused-disable-directives --max-warnings 0"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "bcrypt": "^5.1.0",
+        "connect-session-sequelize": "^7.1.7",
+        "cookie-parser": "^1.4.6",
+        "cors": "^2.8.5",
+        "dotenv": "^16.3.1",
+        "express": "^4.18.2",
+        "express-session": "^1.17.3",
+        "morgan": "^1.10.0",
+        "nodemon": "^2.0.22",
+        "pg": "^8.11.1",
+        "pg-hstore": "^2.3.4",
+        "react": "^18.2.0",
+        "sequelize": "^6.32.1",
+        "socket.io": "^4.7.1"
+    },
+    "devDependencies": {
+        "eslint": "^8.46.0",
+        "eslint-config-airbnb": "^19.0.4",
+        "eslint-config-prettier": "^8.9.0",
+        "eslint-plugin-react": "^7.33.1",
+        "prettier": "^3.0.0"
+    },
+    "prettier": {
```

Comment:
It would look more organized if prettier config lived in its own file 
https://github.com/BriptimusPrimus/sorted-list/blob/main/.prettierrc.js